pkg/scheduler/testing/framework/fake_extender.go

```diff
@@ -236,8 +238,8 @@ func (f *FakeExtender) selectVictimsOnNodeByExtender(pod *v1.Pod, node *v1.Node)
 
 	var potentialVictims []*v1.Pod
 
-	removePod := func(rp *v1.Pod) {
-		nodeInfoCopy.RemovePod(rp)
+	removePod := func(rp *v1.Pod) error {
+		return nodeInfoCopy.RemovePod(logger, rp)
```

Comment:
Checked err for golangci-lint
```
ERROR: pkg/scheduler/scheduler_test.go:286:13: Error return value of `queue.Pop` is not checked (errcheck)
ERROR: 			queue.Pop(logger)
ERROR: 			         ^
ERROR: pkg/scheduler/testing/framework/fake_extender.go:242:25: Error return value of `nodeInfoCopy.RemovePod` is not checked (errcheck)
ERROR: 		nodeInfoCopy.RemovePod(logger, rp)
```
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/120933/pull-kubernetes-linter-hints/1712055285536788480